content_store_handler.go

```diff
@@ -52,20 +45,12 @@ func (cs *ContentStoreHandler) PutContentItem(w http.ResponseWriter, r *http.Req
 		return
 	}
 
-	if !cs.registerWithURLArbiter(path, contentStoreRequest.PublishingApp, w) {
+	if !cs.registerWithURLArbiter(contentStoreRequest.BasePath, contentStoreRequest.PublishingApp, w) {
```

Comment:
The first option would mean that a mismatched base_path would be registered with url-arbiter before being rejected by content-store. The second means adding extra knowledge and behaviour to this app.

I'm leaning towards the latter.